popup/popup.js

```diff
@@ -39,11 +39,12 @@ chrome.storage.local.get(["websiteUrl"]).then(({ websiteUrl }) => {
 			document.querySelector("#session-image").src = user.image;
 			document.querySelector("#session-name").innerHTML = user.name;
 			document.querySelector("#session-email").innerHTML = user.email;
-
+			const cookie = await chrome.cookies.get({ url: websiteUrl, name: '__Secure-next-auth.session-token' });
 			chrome.storage.local.set({
 				userId: user.id,
 				userName: user.name,
-				userImage: user.image
+				userImage: user.image,
+				token: cookie.value,
```

Comment:
And when you implement the `catch` clause, you will have to handle `undefined` cookie here.